engine/src/main/java/org/terasology/math/IntegerRange.java

```diff
@@ -33,6 +33,11 @@ public void addNumbers(int from, int to) {
         }
     }
 
+    @Override
+    public Iterator<Integer> iterator() {
+        return createIterator();
+    }
+
     public Iterator<Integer> createIterator() {
```

Comment:
Is there a reason for having two identical methods with similar names next to each other? If not, I'd remove `createIterator` and fix the calling code. There shouldn't be too many hits.